packages/cli/generators/example/index.js

```diff
@@ -116,8 +116,8 @@ module.exports = class extends BaseGenerator {
     return super.install();
   }
 
-  end() {
-    if (!super.end()) return false;
+  async end() {
+    await super.end();
```

Comment:
I am surprised `super.end()` is causing linting problems in `example/index.js` only. I'd like to review other generators to see how are they implementing `end` and check if we can unify the code.

Reply:
Perhaps I should do like the others, and check `if (this.shouldExit()) {`

![Screenshot 2019-09-16 10 38 18](https://user-images.githubusercontent.com/911799/64931917-27f6d200-d86e-11e9-804f-42c18c136b44.png)

Reply:
I checked other generators, this change LGTM.